app/src/display/main.c

```diff
@@ -91,7 +91,8 @@ int zmk_display_is_initialized() { return initialized; }
 static void initialize_theme() {
 #if IS_ENABLED(CONFIG_LV_USE_THEME_MONO)
     lv_disp_t *disp = lv_disp_get_default();
-    lv_theme_t *theme = lv_theme_mono_init(disp, false, CONFIG_LV_FONT_DEFAULT);
+    const bool invert = IS_ENABLED(CONFIG_ZMK_DISPLAY_INVERT);
+    lv_theme_t *theme = lv_theme_mono_init(disp, invert, CONFIG_LV_FONT_DEFAULT);
```

Comment:
Do we really need to pull this out to a local?

```suggestion
    lv_theme_t *theme = lv_theme_mono_init(disp, IS_ENABLED(CONFIG_ZMK_DISPLAY_INVERT), CONFIG_LV_FONT_DEFAULT);
```

Reply:
Yeah, that works just as well, committed the change.